stories/core/InformationPanel.stories.tsx

```diff
@@ -102,7 +102,34 @@ export const Basic: Story<InformationPanelProps> = (args) => {
           <Text variant='subheading'>Row {openRowIndex ?? 0}</Text>
         </InformationPanelHeader>
         <InformationPanelBody>
-          <Text>{lorem100}</Text>
+          <InformationPanelContent responsiveBreakpoint={250}>
+            <Label htmlFor='name-input'>File name</Label>
+            <Input
+              size='small'
+              id='name-input'
+              defaultValue={`Row ${openRowIndex ?? 0}`}
+              readOnly
+            />
+
+            <Label htmlFor='author-input'>Author</Label>
+            <Input
+              size='small'
+              id='author-input'
+              defaultValue='DJ Terry'
+              readOnly
+            />
+
+            <Label htmlFor='year-input'>Year</Label>
+            <Input size='small' id='year-input' defaultValue='2021' readOnly />
+
+            <Label htmlFor='path-input'>Path</Label>
+            <Input
+              size='small'
+              id='path-input'
+              defaultValue='/Shared/Music/'
+              readOnly
+            />
+          </InformationPanelContent>
```

Comment:
Can you also add splitter and sub-title? So users could see the proper example.
![image](https://user-images.githubusercontent.com/36186912/142416230-554d9c14-76c7-4020-9c05-86665327a563.png)

Reply:
That "proper example" has tons of overrides, which I'm not sure I should add.

1. readonly inputs have border hidden
2. `hr` has custom margin, color and thickness

Reply:
Oh, I just meant the `<hr>` so you just need padding to it as we already provide color from CSS and then "Attributes" which is just leading text.

Reply:
Update story too

Reply:
>  we already provide color from CSS

That color is not enough because it is relying on border. This is what I had to do to make it look the same as our CSS example.
![image](https://user-images.githubusercontent.com/9084735/143889164-501d49f0-1b2d-4830-a8b2-befdd72b6941.png)

Without those overrides, it's too thick.
![image](https://user-images.githubusercontent.com/9084735/143889368-7b0e8a7c-3e5a-4666-8460-ba02fb8e4f1a.png)